src/group-configurations/data/api.js

```diff
@@ -69,3 +69,47 @@ export async function deleteContentGroup(courseId, parentGroupId, groupId) {
 
   return camelCaseObject(data);
 }
+
+/**
+ * Create new experiment configuration for course.
+ * @param {string} courseId
+ * @param {object} configuration
+ * @returns {Promise<Object>}
+ */
+export async function createExperimentConfiguration(courseId, configuration) {
```

Comment:
[question] Do we delete configuration or group? In case if we delete a group it requires renaming to `createExperimentGroup`. For other functions the same question.

Reply:
We remove exactly configuration.
There are types **content group**, **experiment configuration** with actions.
That's what I called such them